src/Recipe.hs

```diff
@@ -0,0 +1,132 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE DeriveGeneric #-}
+
+module Recipe
+  ( createRecipeTable,
+    Ingredient (..),
+    Recipe (..),
+    RecipeID,
+    RecipeWithID (..),
+    handleGetAllRecipes,
+    handleGetRecipe,
+    handleAddRecipe,
+    getAllRecipes,
+    getRecipe,
+    addRecipe,
```

Comment:
es sollten eigentlich nur die handler und DB tables exportiert werden.

Reply:
Exports of "getAllRecipes, getRecipe, addRecipe" removed in 61808d2e21d33e8bbea750823776691997934ac5